docs.cask.co/scripts/builder.py

```diff
@@ -323,16 +341,22 @@ def converted_line(line):
     return line_list[0:4], gcse
 
 def write_js_versions(target):
+    files = []
     target_dir = os.path.dirname(target)
-    version = os.path.join(target_dir, 'version')
+    if get_json_versions():
+        files.append((target, get_json_versions()))
+    if get_current_version():
+        files.append((os.path.join(target_dir, 'version'), get_current_version()))
+    if get_develop_version():
+        files.append((os.path.join(target_dir, 'develop'), get_develop_version()))
```

Comment:
Set `version` and `develop` as variables, then check/assign using those variables, so you don't run the parsing functions twice.